tests/schemas/invalid/test_invalid_answer_comparison_id.json

```diff
@@ -42,7 +42,7 @@
                     "type": "Number"
                   }
                 ],
-                "description": "",
+                "description": "Description",
```

Comment:
I don't think description is required on questions - given that there's nothing useful here it may as well be removed?

Reply:
I've removed all the descriptions

Reply:
Except the one I commented on? 😄

Reply:
Not sure how I missed it, but should be fixed now